overlays/ps-cv-ocr/Pipfile

```diff
@@ -6,6 +6,11 @@ name = "pypi"
 [packages]
 opencv-python = "*"
 pytesseract = "*"
+pillow = "*"
+scikit-image = "*"
+numpy = "*"
```

Comment:
we actually need to add all package from data science generic image in this case, as we are not building from the base image.